media-gfx/kuickshow/kuickshow-0.9.1-r2.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+KDE_LINGUAS="af ar be bg bn br ca ca@valencia cs cy da de el en_GB eo es et eu
+fa fi fr ga gl he hi hne hr hu is it ja km lt lv mai mk ms nb nds ne nl nn oc
+pa pl pt pt_BR ro ru se sk sl sv ta tg th tr uk uz uz@cyrillic vi xh zh_CN
+zh_TW"
+KDE_DOC_DIRS="doc doc-translations/%lingua_${PN}"
+inherit kde4-base
+
+KDE_VERSION=4.4.0
+MY_P=${P}-kde${KDE_VERSION}
+
+DESCRIPTION="KDE program to view images"
+HOMEPAGE="https://userbase.kde.org/KuickShow"
+SRC_URI="mirror://kde/stable/${KDE_VERSION}/src/extragear/${MY_P}.tar.bz2"
```

Comment:
you can find the tarball now at mirror://kde/Attic/4.4.0/src/extragear/${MY_P}.tar.bz2